src/lib/components/ListItemAction.svelte

```diff
@@ -0,0 +1,55 @@
+<div class="container">
```

Comment:
If this component represent list and items, should `ul` and `li` be used to enforce the semantic?

Reply:
Good question. I wasn't sure about it because I didn't enforce a parent that is `ul`.

How about adding the tagName as a prop?

Reply:
I'm confuse now, does this component build the list and the items or only one item?

If only item, then I guess we can ignore my question but, maybe should we rename the component `ItemAction`?

Reply:
Just one item.

I see the confusion. I'll rename but I'll keep the tagName as prop. It can be useful because this component can be used inside lists.

Reply:
done